### ISSUE TITLE
core.inviteAddressWithParams(remoteAddress, params) getting value null for outgoing call method 

### ISSUE THREAD

**Issue by PrarthanaBhatt** [BelledonneCommunications/linphone-android#1846]
Hi,

We are able to get register, for outgoing call we are getting core.inviteAddressWithParams(remoteAddress, params)  as null.

We are using below function from kotlin documentation.

**Link we referred**

https://gitlab.linphone.org/BC/public/tutorials/-/tree/master/android/kotlin/4-OutgoingCall/app/src/main/java/org/linphone/outgoingcall

**Function which we tried,**
    private fun outgoingCall (remoteSipUri: String, context: Context) : String? {

        val remoteAddress = Factory.instance().createAddress(remoteSipUri)
        remoteAddress ?: return null 
        val factory = Factory.instance()
        factory.setLogCollectionPath(context.filesDir.absolutePath)
        factory.enableLogCollection(LogCollectionState.Enabled)
        factory.setLoggerDomain("Drisk")
        factory.enableLogcatLogs(true)
        factory.loggingService.setLogLevel(LogLevel.Message)
        val core: Core = factory.createCore(null, null, this)
        val params = core.createCallParams(null)
        params ?: return null // Same for params
        params.mediaEncryption = MediaEncryption.None
        if (remoteAddress != null) {
            core.inviteAddressWithParams(remoteAddress, params)
        }
    }

**Sharing logs that we are getting.**

V/AutofillManager(22262): requestHideFillUi(null): anchor = null
W/FactoryImpl(22262): Unable to load optional library ffmpeg-linphone: dlopen failed: library "libffmpeg-linphone.so" not found
I/demo   (22262): Initializing LinphoneCore 5.2.18
I/demo   (22262): Sal nat helper [enabled]
I/demo   (22262): ==== Phone information dump ====
I/demo   (22262): DEVICE=OP515BL1
I/demo   (22262): MODEL=DN2101
I/demo   (22262): MANUFACTURER=OnePlus
I/demo   (22262): ANDROID SDK=31
I/demo   (22262): PERFORMANCE CLASS=0
I/demo   (22262): ABIs=arm64-v8a, armeabi-v7a, armeabi
I/demo   (22262): =========================================
I/demo   (22262): ==== Linphone SDK information dump ====
I/demo   (22262): VERSION=5.2.18
I/demo   (22262): BRANCH=tags/5.2.18^0
I/demo   (22262): PLUGINS=libmsandroidcamera2.so, libmswebrtc.so, libmsaaudio.so
I/demo   (22262): PACKAGE=org.linphone.core
I/demo   (22262): BUILD TYPE=release
I/demo   (22262): =========================================
I/demo   (22262): ==== Previous exit reason information dump ====
I/demo   (22262): REASON=User requested
I/demo   (22262): TIMESTAMP=07-02-2023 16:01:47
I/demo   (22262): DESCRIPTION=stop com.example.demodue to from pid 26938|J:0
I/demo   (22262): =========================================
I/demo   (22262): ==== Previous exit reason information dump ====
I/demo   (22262): REASON=User requested
I/demo   (22262): TIMESTAMP=07-02-2023 15:55:27
I/demo   (22262): DESCRIPTION=stop com.example.demodue to from pid 24669|J:850
I/demo   (22262): =========================================
I/demo   (22262): ==== Previous exit reason information dump ====
I/demo   (22262): REASON=User requested
I/demo   (22262): TIMESTAMP=07-02-2023 15:55:18
I/demo   (22262): DESCRIPTION=stop com.example.demodue to from pid 24155|J:0
I/demo   (22262): =========================================
I/demo   (22262): ==== Previous exit reason information dump ====
I/demo   (22262): REASON=User requested
I/demo   (22262): TIMESTAMP=07-02-2023 15:49:26
I/demo   (22262): DESCRIPTION=stop com.example.demodue to from pid 21869|J:850
I/demo   (22262): =========================================
I/demo   (22262): ==== Previous exit reason information dump ====
I/demo   (22262): REASON=User requested
I/demo   (22262): TIMESTAMP=07-02-2023 15:12:59
I/demo   (22262): DESCRIPTION=stop com.example.demodue to from pid 7466|J:0
I/demo   (22262): =========================================
I/demo   (22262): [Headset] Headset receiver created
I/demo   (22262): [Audio Helper] Helper created
I/demo   (22262): [Bluetooth] Adapter found
I/AudioManager(22262): In isBluetoothScoAvailableOffCall(), calling appilication: com.example.demo, return value: true
I/demo   (22262): [Bluetooth] SCO available off call, continue
I/demo   (22262): [Bluetooth] Bluetooth receiver created
I/demo   (22262): [Bluetooth] Bluetooth broadcast receiver registered
I/demo   (22262): [Bluetooth] Bluetooth helper created
I/demo   (22262): [Core Manager] Device orientation is 0 (raw value is 0)
I/demo   (22262): [Core Manager] Registering shutdown receiver
I/demo   (22262): [Core Manager] Found a service that herits from org.linphone.core.tools.service.CoreService: org.linphone.core.tools.service.CoreService
I/demo   (22262): [Core Manager] Ready
I/demo   (22262): [Android Platform Helper] CoreManager is fully initialised.
I/demo   (22262): [Device] hasLowLatencyFeature: true, hasProFeature: false
I/demo   (22262): [Device] Output frames per buffer: 256, output sample rate: 48000.
I/demo   (22262): [Platform Helper] Created, wifi only mode is disabled
I/demo   (22262): [Platform Helper] Starting copy from assets to application files directory
I/demo   (22262): [Platform Helper] Installing Resource cpim_grammar
I/demo   (22262): [Platform Helper] Installing Resource ics_grammar
I/demo   (22262): [Platform Helper] Installing Resource identity_grammar
I/demo   (22262): [Platform Helper] Installing Resource sdp_grammar
I/demo   (22262): [Platform Helper] Installing Resource vcard_grammar
I/demo   (22262): [Platform Helper] Installing Resource nowebcamcif.jpg
I/demo   (22262): [Platform Helper] Installing Resource rootca.pem
I/demo   (22262): [Platform Helper] Resource dont_wait_too_long.mkv already installed, skipping...
I/demo   (22262): [Platform Helper] Resource hello16000.wav already installed, skipping...
I/demo   (22262): [Platform Helper] Resource hello8000.wav already installed, skipping...
I/demo   (22262): [Platform Helper] Resource incoming_chat.wav already installed, skipping...
I/demo   (22262): [Platform Helper] Resource ringback.wav already installed, skipping...
I/demo   (22262): [Platform Helper] Resource four_hands_together.mkv already installed, skipping...
I/demo   (22262): [Platform Helper] Resource house_keeping.mkv already installed, skipping...
I/demo   (22262): [Platform Helper] Resource its_a_game.mkv already installed, skipping...
I/demo   (22262): [Platform Helper] Resource leaving_dreams.mkv already installed, skipping...
I/demo   (22262): [Platform Helper] Resource notes_of_the_optimistic.mkv already installed, skipping...
I/demo   (22262): [Platform Helper] Resource oldphone-mono.wav already installed, skipping...
I/demo   (22262): [Platform Helper] Resource soft_as_snow.mkv already installed, skipping...
I/demo   (22262): [Platform Helper] Resource silence.mkv already installed, skipping...
I/demo   (22262): [Platform Helper] Resource toy-mono.wav already installed, skipping...
I/demo   (22262): [Platform Helper] Copy from assets done
W/demo   (22262): [Platform Helper] Device is in bucket STANDBY_BUCKET_ACTIVE
D/CompatibilityChangeReporter(22262): Compat change id reported: 186082280; UID 10687; state: DISABLED
I/demo   (22262): [Platform Helper] [Network Manager 26] Found DNS host 2405:200:800::11 from mobile network
I/demo   (22262): [Platform Helper] [Network Manager 26] Found DNS host 2405:200:800::11 from active network MOBILE
I/demo   (22262): [Platform Helper] DNS servers list updated
I/demo   (22262): belle_sip_set_jvm(): initialization succeed
I/demo   (22262): bellesip_wake_lock_init(): initialization succeed
I/demo   (22262): [Android Platform Helper] AndroidPlatformHelper is fully initialised.
I/demo   (22262): Mediastreamer2 factory 5.2.0 (git: 5.2.16) initialized.
I/demo   (22262): CPU count set to 8
I/demo   (22262): ms_factory_init() done: platform_tags=linux,android,x86,embedded
I/demo   (22262): srtp init
I/demo   (22262): SDK version [31] detected
D/om.example.demo(22262): DLEAK reply value=48
D/MediaCodecList(22262): DLEAK matchIndex=48
D/MediaCodecList(22262): codecHandlesFormat: no format, so no extra checks
D/MediaCodecList(22262): DLEAK matching 'c2.mtk.avc.encoder'
D/om.example.demo(22262): DLEAK reply value=49
D/MediaCodecList(22262): DLEAK matchIndex=49
D/MediaCodecList(22262): codecHandlesFormat: no format, so no extra checks
D/MediaCodecList(22262): DLEAK matching 'c2.mtk.avc.encoder.secure'
D/om.example.demo(22262): DLEAK reply value=66
D/MediaCodecList(22262): DLEAK matchIndex=66
D/MediaCodecList(22262): codecHandlesFormat: no format, so no extra checks
D/MediaCodecList(22262): DLEAK matching 'c2.android.avc.encoder'
D/om.example.demo(22262): DLEAK reply value=-2
D/MediaCodecList(22262): DLEAK matchIndex=18446744073709551614
E/Parcel  (22262): Reading a NULL string not supported here.
D/OplusCCodec(22262): OplusCCodec [129]: (0xb40000764806d000)
W/libc    (22262): Access denied finding property "vendor.oplus.media.vpp.stutter"
D/ListServiceUtils(22262): mListServiceUtils::init CallingPid 22262
D/ListServiceUtils(22262): mListServiceUtils::init this 0xb40000759df55f00
V/ListServiceUtils(22262): checkInListByUid module:sr-osie-whitelist uid:10687 cost 128us return (null)
D/CCodec  (22262): allocate(c2.mtk.avc.encoder)
I/Codec2Client(22262): Available Codec2 services: "default" "software"
I/CCodec  (22262): setting up 'default' as default (vendor) store
I/CCodec  (22262): Created component [c2.mtk.avc.encoder]
D/CCodecConfig(22262): read media type: video/avc
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.max-count.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.subscribed-indices.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: input.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.buffers.pool-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.pool-ids.values
D/ReflectedParamUpdater(22262): ignored struct field coding.gop.values
D/ReflectedParamUpdater(22262): ignored struct field coding.qp.values
D/CCodecConfig(22262): ignoring local param raw.color (0xc2001809) as it is already supported
I/CCodecConfig(22262): query failed after returning 22 values (BAD_INDEX)
D/CCodecConfig(22262): c2 config diff is Dict {
D/CCodecConfig(22262):   c2::u32 algo.bitrate-mode.value = 3
D/CCodecConfig(22262):   c2::u32 algo.delay.value = 8
D/CCodecConfig(22262):   c2::i32 algo.priority.value = 0
D/CCodecConfig(22262):   c2::u32 algo.quality.value = 80
D/CCodecConfig(22262):   c2::float algo.rate.value = 0
D/CCodecConfig(22262):   c2::u32 coded.bitrate.value = 64000
D/CCodecConfig(22262):   c2::float coded.frame-rate.value = 30
D/CCodecConfig(22262):   c2::u32 coded.pl.level = 20495
D/CCodecConfig(22262):   c2::u32 coded.pl.profile = 20484
D/CCodecConfig(22262):   c2::u32 coded.tile-layout.columns = 2
D/CCodecConfig(22262):   c2::u32 coded.tile-layout.order = 0
D/CCodecConfig(22262):   c2::u32 coded.tile-layout.rows = 1
D/CCodecConfig(22262):   c2::u32 coded.tile-layout.tile.height = 256
D/CCodecConfig(22262):   c2::u32 coded.tile-layout.tile.width = 256
D/CCodecConfig(22262):   c2::u32 coded.vui.color.matrix = 0
D/CCodecConfig(22262):   c2::u32 coded.vui.color.primaries = 0
D/CCodecConfig(22262):   c2::u32 coded.vui.color.range = 0
D/CCodecConfig(22262):   c2::u32 coded.vui.color.transfer = 0
D/CCodecConfig(22262):   c2::u32 coding.intra-refresh.mode = 0
D/CCodecConfig(22262):   c2::float coding.intra-refresh.period = 0
D/CCodecConfig(22262):   c2::u32 coding.request-sync-frame.value = 0
D/CCodecConfig(22262):   c2::i64 coding.sync-frame-interval.value = 1000000
D/CCodecConfig(22262):   Buffer coding.temporal-layering = {
D/CCodecConfig(22262):     00000000:  10 00 00 00 07 20 01 52  00 00 00 00 00 00 00 00  ..... .R........
D/CCodecConfig(22262):   }
D/CCodecConfig(22262):   c2::u32 input.de
W/ColorUtils(22262): expected specified color aspects (0:0:0:0)
I/demo   (22262): Codec video/avc successfully created.
D/OplusCCodec(22262): initiateShutdown [565]: (0xb40000764806d000) keepComponentAllocated=0
I/hw-BpHwBinder(22262): onLastStrongRef automatically unlinking death recipients
D/om.example.demo(22262): DLEAK reply value=46
D/MediaCodecList(22262): DLEAK matchIndex=46
D/MediaCodecList(22262): codecHandlesFormat: no format, so no extra checks
D/MediaCodecList(22262): DLEAK matching 'c2.mtk.avc.decoder'
D/om.example.demo(22262): DLEAK reply value=65
D/OplusCCodec(22262): ~OplusCCodec [144]: (0xb40000764806d000)
D/MediaCodecList(22262): DLEAK matchIndex=65
D/MediaCodecList(22262): codecHandlesFormat: no format, so no extra checks
D/MediaCodecList(22262): DLEAK matching 'c2.android.avc.decoder'
D/om.example.demo(22262): DLEAK reply value=-2
D/MediaCodecList(22262): DLEAK matchIndex=18446744073709551614
E/Parcel  (22262): Reading a NULL string not supported here.
D/OplusCCodec(22262): OplusCCodec [129]: (0xb4000075e0054440)
W/libc    (22262): Access denied finding property "vendor.oplus.media.vpp.stutter"
V/ListServiceUtils(22262): checkInListByUid module:sr-osie-whitelist uid:10687 cost 102us return (null)
D/CCodec  (22262): allocate(c2.mtk.avc.decoder)
I/CCodec  (22262): setting up 'default' as default (vendor) store
I/CCodec  (22262): Created component [c2.mtk.avc.decoder]
D/CCodecConfig(22262): read media type: video/avc
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.max-count.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.subscribed-indices.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: input.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.buffers.pool-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.pool-ids.values
D/ReflectedParamUpdater(22262): ignored struct field coded.color-format.locations
D/CCodecConfig(22262): ignoring local param raw.size (0xd2001800) as it is already supported
D/CCodecConfig(22262): ignoring local param raw.crop (0xd2001801) as it is already supported
D/CCodecConfig(22262): ignoring local param default.color (0x5200180b) as it is already supported
D/CCodecConfig(22262): ignoring local param raw.hdr-static-info (0xd200180a) as it is already supported
I/CCodecConfig(22262): query failed after returning 17 values (BAD_INDEX)
D/CCodecConfig(22262): c2 config diff is Dict {
D/CCodecConfig(22262):   c2::i32 algo.priority.value = 1
D/CCodecConfig(22262):   c2::float algo.rate.value = 0
D/CCodecConfig(22262):   c2::float coded.frame-rate.value = 30
D/CCodecConfig(22262):   c2::u32 coded.pl.level = 20495
D/CCodecConfig(22262):   c2::u32 coded.pl.profile = 20481
D/CCodecConfig(22262):   c2::u32 coded.vui.color.matrix = 0
D/CCodecConfig(22262):   c2::u32 coded.vui.color.primaries = 0
D/CCodecConfig(22262):   c2::u32 coded.vui.color.range = 2
D/CCodecConfig(22262):   c2::u32 coded.vui.color.transfer = 0
D/CCodecConfig(22262):   c2::u32 default.color.matrix = 0
D/CCodecConfig(22262):   c2::u32 default.color.primaries = 0
D/CCodecConfig(22262):   c2::u32 default.color.range = 0
D/CCodecConfig(22262):   c2::u32 default.color.transfer = 0
D/CCodecConfig(22262):   c2::u32 input.buffers.max-size.value = 2097152
D/CCodecConfig(22262):   c2::u32 input.delay.value = 4
D/CCodecConfig(22262):   string input.media-type.value = "video/avc"
D/CCodecConfig(22262):   c2::u32 output.delay.value = 12
D/CCodecConfig(22262):   string output.media-type.value = "video/raw"
D/CCodecConfig(22262):   c2::u32 raw.color.matrix = 0
D/CCodecConfig(22262):   c2::u32 raw.color.primaries = 0
D/CCodecConfig(22262):   c2::u32 raw.color.range = 2
D/CCodecConfig(22262):   c2::u32 raw.color.transfer = 0
D/CCodecConfig(22262):   c2::u32 raw.crop.height = 239
D/CCodecConfig(22262):   c2::u32 raw.crop.left = 0
D/CCodecConfig(22262):   c2::u32 raw.crop.top = 0
D/CCodecConfig(22262):   c2::u32 raw.crop.width = 319
D/CCodecConfig(22262):   c2::float raw.hdr-static-info.mastering.blue.x = 0
D/CCodecConfig(22262):   c2::float raw.hdr-s
W/ColorUtils(22262): expected specified color aspects (2:0:0:0)
I/demo   (22262): Codec video/avc successfully created.
D/OplusCCodec(22262): initiateShutdown [565]: (0xb4000075e0054440) keepComponentAllocated=0
I/demo   (22262): MediaCodec: 'video/avc' format supported
D/om.example.demo(22262): DLEAK reply value=56
D/MediaCodecList(22262): DLEAK matchIndex=56
D/MediaCodecList(22262): codecHandlesFormat: no format, so no extra checks
D/MediaCodecList(22262): DLEAK matching 'c2.mtk.hevc.encoder'
D/om.example.demo(22262): DLEAK reply value=-2
D/MediaCodecList(22262): DLEAK matchIndex=18446744073709551614
D/MediaCodec(22262): MediaCodec::reclaim(0xb4000075dfe46600) c2.mtk.avc.decoder
D/MediaCodec(22262): MediaCodec looper is gone, skip reclaim
D/OplusCCodec(22262): ~OplusCCodec [144]: (0xb4000075e0054440)
W/AMessage(22262): failed to deliver message as target handler 5 is gone.
I/hw-BpHwBinder(22262): onLastStrongRef automatically unlinking death recipients
D/OplusCCodec(22262): OplusCCodec [129]: (0xb4000075e0054440)
W/libc    (22262): Access denied finding property "vendor.oplus.media.vpp.stutter"
V/ListServiceUtils(22262): checkInListByUid module:sr-osie-whitelist uid:10687 cost 202us return (null)
D/CCodec  (22262): allocate(c2.mtk.hevc.encoder)
I/CCodec  (22262): setting up 'default' as default (vendor) store
I/CCodec  (22262): Created component [c2.mtk.hevc.encoder]
D/CCodecConfig(22262): read media type: video/hevc
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.max-count.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.subscribed-indices.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: input.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.buffers.pool-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.pool-ids.values
D/ReflectedParamUpdater(22262): ignored struct field coding.gop.values
D/ReflectedParamUpdater(22262): ignored struct field coding.qp.values
D/CCodecConfig(22262): ignoring local param raw.color (0xc2001809) as it is already supported
I/CCodecConfig(22262): query failed after returning 22 values (BAD_INDEX)
D/CCodecConfig(22262): c2 config diff is Dict {
D/CCodecConfig(22262):   c2::u32 algo.bitrate-mode.value = 3
D/CCodecConfig(22262):   c2::u32 algo.delay.value = 8
D/CCodecConfig(22262):   c2::i32 algo.priority.value = 0
D/CCodecConfig(22262):   c2::u32 algo.quality.value = 80
D/CCodecConfig(22262):   c2::float algo.rate.value = 0
D/CCodecConfig(22262):   c2::u32 coded.bitrate.value = 64000
D/CCodecConfig(22262):   c2::float coded.frame-rate.value = 30
D/CCodecConfig(22262):   c2::u32 coded.pl.level = 24584
D/CCodecConfig(22262):   c2::u32 coded.pl.profile = 24576
D/CCodecConfig(22262):   c2::u32 coded.tile-layout.columns = 2
D/CCodecConfig(22262):   c2::u32 coded.tile-layout.order = 0
D/CCodecConfig(22262):   c2::u32 coded.tile-layout.rows = 1
D/CCodecConfig(22262):   c2::u32 coded.tile-layout.tile.height = 256
D/CCodecConfig(22262):   c2::u32 coded.tile-layout.tile.width = 256
D/CCodecConfig(22262):   c2::u32 coded.vui.color.matrix = 0
D/CCodecConfig(22262):   c2::u32 coded.vui.color.primaries = 0
D/CCodecConfig(22262):   c2::u32 coded.vui.color.range = 0
D/CCodecConfig(22262):   c2::u32 coded.vui.color.transfer = 0
D/CCodecConfig(22262):   c2::u32 coding.intra-refresh.mode = 0
D/CCodecConfig(22262):   c2::float coding.intra-refresh.period = 0
D/CCodecConfig(22262):   c2::u32 coding.request-sync-frame.value = 0
D/CCodecConfig(22262):   c2::i64 coding.sync-frame-interval.value = 1000000
D/CCodecConfig(22262):   Buffer coding.temporal-layering = {
D/CCodecConfig(22262):     00000000:  10 00 00 00 07 20 01 52  00 00 00 00 00 00 00 00  ..... .R........
D/CCodecConfig(22262):   }
D/CCodecConfig(22262):   c2::u32 input.de
W/ColorUtils(22262): expected specified color aspects (0:0:0:0)
I/demo   (22262): Codec video/hevc successfully created.
D/OplusCCodec(22262): initiateShutdown [565]: (0xb4000075e0054440) keepComponentAllocated=0
D/om.example.demo(22262): DLEAK reply value=54
D/MediaCodecList(22262): DLEAK matchIndex=54
D/MediaCodecList(22262): codecHandlesFormat: no format, so no extra checks
D/MediaCodecList(22262): DLEAK matching 'c2.mtk.hevc.decoder'
D/om.example.demo(22262): DLEAK reply value=69
D/MediaCodecList(22262): DLEAK matchIndex=69
D/MediaCodecList(22262): codecHandlesFormat: no format, so no extra checks
D/MediaCodecList(22262): DLEAK matching 'c2.android.hevc.decoder'
D/om.example.demo(22262): DLEAK reply value=-2
D/MediaCodecList(22262): DLEAK matchIndex=18446744073709551614
I/hw-BpHwBinder(22262): onLastStrongRef automatically unlinking death recipients
D/OplusCCodec(22262): ~OplusCCodec [144]: (0xb4000075e0054440)
E/Parcel  (22262): Reading a NULL string not supported here.
D/OplusCCodec(22262): OplusCCodec [129]: (0xb40000764806d000)
W/libc    (22262): Access denied finding property "vendor.oplus.media.vpp.stutter"
V/ListServiceUtils(22262): checkInListByUid module:sr-osie-whitelist uid:10687 cost 164us return (null)
D/CCodec  (22262): allocate(c2.mtk.hevc.decoder)
I/CCodec  (22262): setting up 'default' as default (vendor) store
I/CCodec  (22262): Created component [c2.mtk.hevc.decoder]
D/CCodecConfig(22262): read media type: video/hevc
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.max-count.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.subscribed-indices.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: input.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.allocator-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: output.buffers.pool-ids.values
D/ReflectedParamUpdater(22262): extent() != 1 for single value type: algo.buffers.pool-ids.values
D/ReflectedParamUpdater(22262): ignored struct field coded.color-format.locations
D/CCodecConfig(22262): ignoring local param raw.size (0xd2001800) as it is already supported
D/CCodecConfig(22262): ignoring local param raw.crop (0xd2001801) as it is already supported
D/CCodecConfig(22262): ignoring local param default.color (0x5200180b) as it is already supported
D/CCodecConfig(22262): ignoring local param raw.hdr-static-info (0xd200180a) as it is already supported
I/CCodecConfig(22262): query failed after returning 17 values (BAD_INDEX)
D/CCodecConfig(22262): c2 config diff is Dict {
D/CCodecConfig(22262):   c2::i32 algo.priority.value = 1
D/CCodecConfig(22262):   c2::float algo.rate.value = 0
D/CCodecConfig(22262):   c2::float coded.frame-rate.value = 30
D/CCodecConfig(22262):   c2::u32 coded.pl.level = 24584
D/CCodecConfig(22262):   c2::u32 coded.pl.profile = 20481
D/CCodecConfig(22262):   c2::u32 coded.vui.color.matrix = 0
D/CCodecConfig(22262):   c2::u32 coded.vui.color.primaries = 0
D/CCodecConfig(22262):   c2::u32 coded.vui.color.range = 2
D/CCodecConfig(22262):   c2::u32 coded.vui.color.transfer = 0
D/CCodecConfig(22262):   c2::u32 default.color.matrix = 0
D/CCodecConfig(22262):   c2::u32 default.color.primaries = 0
D/CCodecConfig(22262):   c2::u32 default.color.range = 0
D/CCodecConfig(22262):   c2::u32 default.color.transfer = 0
D/CCodecConfig(22262):   c2::u32 input.buffers.max-size.value = 2097152
D/CCodecConfig(22262):   c2::u32 input.delay.value = 4
D/CCodecConfig(22262):   string input.media-type.value = "video/hevc"
D/CCodecConfig(22262):   c2::u32 output.delay.value = 12
D/CCodecConfig(22262):   string output.media-type.value = "video/raw"
D/CCodecConfig(22262):   c2::u32 raw.color.matrix = 0
D/CCodecConfig(22262):   c2::u32 raw.color.primaries = 0
D/CCodecConfig(22262):   c2::u32 raw.color.range = 2
D/CCodecConfig(22262):   c2::u32 raw.color.transfer = 0
D/CCodecConfig(22262):   c2::u32 raw.crop.height = 239
D/CCodecConfig(22262):   c2::u32 raw.crop.left = 0
D/CCodecConfig(22262):   c2::u32 raw.crop.top = 0
D/CCodecConfig(22262):   c2::u32 raw.crop.width = 319
D/CCodecConfig(22262):   c2::float raw.hdr-static-info.mastering.blue.x = 0
D/CCodecConfig(22262):   c2::float raw.hdr-
W/ColorUtils(22262): expected specified color aspects (2:0:0:0)
I/demo   (22262): Codec video/hevc successfully created.
D/OplusCCodec(22262): initiateShutdown [565]: (0xb40000764806d000) keepComponentAllocated=0
I/demo   (22262): MediaCodec: 'video/hevc' format supported
I/demo   (22262): Registering all soundcard handlers
I/demo   (22262): SDK version [31] detected
I/demo   (22262): Native android sound support not tested on SDK [31], disabled.
I/demo   (22262): [OpenSLES] libOpenSLES correctly loaded, creating OpenSLES MS soundcard
I/hw-BpHwBinder(22262): onLastStrongRef automatically unlinking death recipients
I/AudioEffect(22262): queryNumberEffects(): *numEffects = 17, ret = 0
D/OplusCCodec(22262): ~OplusCCodec [144]: (0xb40000764806d000)
I/demo   (22262): This device (OnePlus/DN2101/mt6893) declares it has a built-in echo canceller.
I/demo   (22262): No information available for [OnePlus/DN2101/mt6893],
I/demo   (22262): Android >= 8, using sound device descriptor.with DEVICE_HAS_BUILTIN_AEC | DEVICE_HAS_BUILTIN_OPENSLES_AEC flags
I/demo   (22262): Sound device information for [OnePlus/DN2101/mt6893] is: builtin=[yes], delay=[0] ms
I/demo   (22262): [Android Audio Utils] Using 48000 for sample rate value
I/demo   (22262): [Android Audio Utils] Using 256 for buffer size value
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [2], type [Built-in earpiece (1)], isSource [false], isSink [true]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [3], type [Built-in speaker (2)], isSource [false], isSink [true]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [13], type [Telephony (18)], isSource [false], isSink [true]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [16], type [Built-in microphone (15)], isSource [true], isSink [false]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [20], type [Telephony (18)], isSource [true], isSink [false]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [17], type [Built-in microphone (15)], isSource [true], isSink [false]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [27], type [Remote Submix (25)], isSource [true], isSink [false]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [22], type [FM tuner (16)], isSource [true], isSink [false]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [23], type [UNEXPECTED (28)], isSource [true], isSink [false]
I/demo   (22262): [OpenSLES] Create soundcards for 9 devices
I/demo   (22262): Card 'openSLES Earpiece: DN2101' added with capabilities [playback]
I/demo   (22262): [OpenSLES] Added card [0xb400007639e42d90]: name [DN2101] device ID [2] type [Earpiece]
I/demo   (22262): Card 'openSLES Speaker: DN2101' added with capabilities [playback]
I/demo   (22262): [OpenSLES] Added card [0xb40000764b11b470]: name [DN2101] device ID [3] type [Speaker]
I/demo   (22262): [OpenSLES] SKipped device with type [Telephony]
I/demo   (22262): [OpenSLES] Adding MS_SND_CARD_CAP_BUILTIN_ECHO_CANCELLER flag to soundcard ([DN2101] device ID [16] type [Microphone]) because of DEVICE_HAS_BUILTIN_OPENSLES_AEC
I/demo   (22262): Card 'openSLES Microphone: DN2101' added with capabilities [capture]
I/demo   (22262): [OpenSLES] Added card [0xb40000758af1be30]: name [DN2101] device ID [16] type [Microphone]
I/demo   (22262): [OpenSLES] SKipped device with type [Telephony]
I/demo   (22262): [OpenSLES] Adding MS_SND_CARD_CAP_BUILTIN_ECHO_CANCELLER flag to soundcard ([DN2101] device ID [17] type [Microphone]) because of DEVICE_HAS_BUILTIN_OPENSLES_AEC
W/demo   (22262): [OpenSLES] Card [0xb4000075e0f1f4f0]: name [DN2101] device ID [17] type [Microphone] not added, considered a duplicate of card [0xb40000758af1be30]: name [DN2101] device ID [16] type [Microphone]
I/demo   (22262): [OpenSLES] Deleting card [0xb4000075e0f1f4f0]: name [DN2101] device ID [17] type [Microphone]
I/demo   (22262): [OpenSLES] Deletion of OpenSLES context [0xb400007588a90080]
E/demo   (22262): [Android Audio Utils] Unknown device type for type ID 25
I/demo   (22262): [OpenSLES] SKipped device with type [Unknown]
E/demo   (22262): [Android Audio Utils] Unknown device type for type ID 16
I/demo   (22262): [OpenSLES] SKipped device with type [Unknown]
E/demo   (22262): [Android Audio Utils] Unknown device type for type ID 28
I/demo   (22262): [OpenSLES] SKipped device with type [Unknown]
I/demo   (22262): SDK version [31] detected
I/AudioEffect(22262): queryNumberEffects(): *numEffects = 17, ret = 0
I/demo   (22262): This device (OnePlus/DN2101/mt6893) declares it has a built-in echo canceller.
I/demo   (22262): No information available for [OnePlus/DN2101/mt6893],
I/demo   (22262): Android >= 8, using sound device descriptor.with DEVICE_HAS_BUILTIN_AEC | DEVICE_HAS_BUILTIN_OPENSLES_AEC flags
I/demo   (22262): Sound device information for [OnePlus/DN2101/mt6893] is: builtin=[yes], delay=[0] ms
I/demo   (22262): Card 'ANDROID SND (deprecated) Unknown: Android Sound card' added with capabilities [capture, playback]
I/demo   (22262): Registering all webcam handlers
I/demo   (22262): Found Android plugin libmsandroidcamera2.so
I/demo   (22262): Found Android plugin libmswebrtc.so
I/demo   (22262): Found Android plugin libmsaaudio.so
W/demo   (22262): [Legacy Capture] Android >= 8.0 detected and msAndroidCamera2 plugin found, disabling legacy capture filter
I/demo   (22262): Webcam StaticImage: Static picture added
I/demo   (22262): MSAndroidDisplay registered.
I/demo   (22262): MSAndroidDisplay (OpenGL ES2) registered (id=81).
I/demo   (22262): ms_factory_init_voip() done
I/demo   (22262): Found Android plugin libmsandroidcamera2.so
I/demo   (22262): Found Android plugin libmswebrtc.so
I/demo   (22262): Found Android plugin libmsaaudio.so
I/demo   (22262): Loading ms plugins from list
I/demo   (22262): Loading plugin libmsandroidcamera2.so...
I/demo   (22262): [Camera2 Capture] libmsandroidcamera2 plugin loaded
I/demo   (22262): [Camera2 Capture] Detecting cameras
I/demo   (22262): [Camera2 Capture] Camera 0 is facing back with angle 90, hardware level is 3
I/demo   (22262): Webcam BackFacingCamera prepended
I/demo   (22262): [Camera2 Capture] Camera 1 is facing front with angle 270, hardware level is full
I/demo   (22262): Webcam FrontFacingCamera prepended
I/demo   (22262): Plugin loaded (libmsandroidcamera2.so)
I/demo   (22262): Loading plugin libmswebrtc.so...
I/demo   (22262): libmswebrtc 1.1.1 plugin loaded, iSAC codec version 3.6.0, iLBC codec version 1.1.1
I/demo   (22262): Plugin loaded (libmswebrtc.so)
I/demo   (22262): Loading plugin libmsaaudio.so...
I/demo   (22262): [AAudio] libmsaaudio plugin loaded
I/demo   (22262): [Android Audio Utils] Using 48000 for sample rate value
I/demo   (22262): [AAudio] libaaudio.so successfully loaded
I/AudioEffect(22262): queryNumberEffects(): *numEffects = 17, ret = 0
I/demo   (22262): This device (OnePlus/DN2101/mt6893) declares it has a built-in echo canceller.
I/demo   (22262): No information available for [OnePlus/DN2101/mt6893],
I/demo   (22262): Android >= 8, using sound device descriptor.with DEVICE_HAS_BUILTIN_AEC | DEVICE_HAS_BUILTIN_OPENSLES_AEC flags
I/demo   (22262): Sound device information for [OnePlus/DN2101/mt6893] is: builtin=[yes], delay=[0] ms
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [2], type [Built-in earpiece (1)], isSource [false], isSink [true]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [3], type [Built-in speaker (2)], isSource [false], isSink [true]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [13], type [Telephony (18)], isSource [false], isSink [true]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [16], type [Built-in microphone (15)], isSource [true], isSink [false]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [20], type [Telephony (18)], isSource [true], isSink [false]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [17], type [Built-in microphone (15)], isSource [true], isSink [false]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [27], type [Remote Submix (25)], isSource [true], isSink [false]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [22], type [FM tuner (16)], isSource [true], isSink [false]
I/demo   (22262): [Audio Manager] Found device: name [DN2101], ID [23], type [UNEXPECTED (28)], isSource [true], isSink [false]
I/demo   (22262): [AAudio] Create soundcards for 9 devices
I/AudioEffect(22262): queryNumberEffects(): *numEffects = 17, ret = 0
I/demo   (22262): This device (OnePlus/DN2101/mt6893) declares it has a built-in echo canceller.
I/demo   (22262): No information available for [OnePlus/DN2101/mt6893],
I/demo   (22262): Android >= 8, using sound device descriptor.with DEVICE_HAS_BUILTIN_AEC | DEVICE_HAS_BUILTIN_OPENSLES_AEC flags
I/demo   (22262): Sound device information for [OnePlus/DN2101/mt6893] is: builtin=[yes], delay=[0] ms
I/demo   (22262): Card 'AAudio Earpiece: DN2101' prepended with capabilities [playback]
I/demo   (22262): [AAudio] Added card with ID: [AAudio Earpiece: DN2101], name: [DN2101], device ID: [2], type: [Earpiece] and capabilities: [2]
I/demo   (22262): Card 'AAudio Speaker: DN2101' prepended with capabilities [playback]
I/demo   (22262): [AAudio] Added card with ID: [AAudio Speaker: DN2101], name: [DN2101], device ID: [3], type: [Speaker] and capabilities: [2]
I/demo   (22262): [AAudio] SKipped device with type [Telephony]
I/demo   (22262): Card 'AAudio Microphone: DN2101' prepended with capabilities [capture]
I/demo   (22262): [AAudio] Added card with ID: [AAudio Microphone: DN2101], name: [DN2101], device ID: [16], type: [Microphone] and capabilities: [5]
I/demo   (22262): [AAudio] SKipped device with type [Telephony]
I/demo   (22262): [AAudio] Card with ID: [(null)], name: [DN2101], device ID: [17], type: [Microphone] and capabilities: [5] not added, considered as duplicate
I/demo   (22262): [AAudio] Deletion of AAudio context [0xb40000758af2f7f0]
E/demo   (22262): [Android Audio Utils] Unknown device type for type ID 25
I/demo   (22262): [AAudio] SKipped device with type [Unknown]
E/demo   (22262): [Android Audio Utils] Unknown device type for type ID 16
I/demo   (22262): [AAudio] SKipped device with type [Unknown]
E/demo   (22262): [Android Audio Utils] Unknown device type for type ID 28
I/demo   (22262): [AAudio] SKipped device with type [Unknown]
I/demo   (22262): [AAudio] Soundcard created
I/demo   (22262): Plugin loaded (libmsaaudio.so)
I/demo   (22262): All plugins in list correctly loaded
I/demo   (22262): Core callbacks [0xb400007648065cb0] registered on core [0xb4000075e007e500]
I/demo   (22262): oRTP-5.2.0 initialized with Atomic protection.
I/demo   (22262): Codec opus/48000 fmtp=[useinbandfec=1] number=-1, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec SILK/16000 fmtp=[] number=-1, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec speex/16000 fmtp=[vbr=on] number=-1, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec speex/8000 fmtp=[vbr=on] number=-1, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec PCMU/8000 fmtp=[] number=0, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec PCMA/8000 fmtp=[] number=8, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec red/1000 fmtp=[] number=-1, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec t140/1000 fmtp=[] number=-1, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec GSM/8000 fmtp=[] number=3, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec G722/8000 fmtp=[] number=9, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec iLBC/8000 fmtp=[mode=30] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec AMR/8000 fmtp=[octet-align=1] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec AMR-WB/16000 fmtp=[octet-align=1] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec G729/8000 fmtp=[annexb=yes] number=18, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec mpeg4-generic/16000 fmtp=[config=F8EE2000; constantDuration=512; indexDeltaLength=3; indexLength=3; mode=AAC-hbr; profile-level-id=76; sizeLength=13; streamType=5] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec mpeg4-generic/22050 fmtp=[config=F8EE2000; constantDuration=512; indexDeltaLength=3; indexLength=3; mode=AAC-hbr; profile-level-id=76; sizeLength=13; streamType=5] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec mpeg4-generic/32000 fmtp=[config=F8E82000; constantDuration=512; indexDeltaLength=3; indexLength=3; mode=AAC-hbr; profile-level-id=76; sizeLength=13; streamType=5] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec mpeg4-generic/44100 fmtp=[config=F8E82000; constantDuration=512; indexDeltaLength=3; indexLength=3; mode=AAC-hbr; profile-level-id=76; sizeLength=13; streamType=5] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec mpeg4-generic/48000 fmtp=[config=F8EE2000; constantDuration=512; indexDeltaLength=3; indexLength=3; mode=AAC-hbr; profile-level-id=76; sizeLength=13; streamType=5] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec iSAC/16000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec speex/32000 fmtp=[vbr=on] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec SILK/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec SILK/12000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec SILK/24000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec G726-16/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec G726-24/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec G726-32/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec G726-40/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec AAL2-G726-16/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec AAL2-G726-24/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec AAL2-G726-32/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec AAL2-G726-40/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec CODEC2/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec BV16/8000 fmtp=[] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec VP8/90000 fmtp=[] number=-1, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec H264/90000 fmtp=[profile-level-id=42801F] number=-1, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec H265/90000 fmtp=[] number=-1, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec MP4V-ES/90000 fmtp=[profile-level-id=3] number=-1, default enablement: 1) added to the list of possible codecs.
I/demo   (22262): Codec H263-1998/90000 fmtp=[CIF=1;QCIF=1] number=-1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec H263/90000 fmtp=[] number=34, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec 1016/8000 fmtp=[] number=1, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec G723/8000 fmtp=[] number=4, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec LPC/8000 fmtp=[] number=7, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec L16/44100 fmtp=[] number=10, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec L16/44100 fmtp=[] number=11, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec CN/8000 fmtp=[] number=13, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec H261/90000 fmtp=[] number=31, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Codec MPV/90000 fmtp=[] number=32, default enablement: 0) added to the list of possible codecs.
I/demo   (22262): Root ca path set to /system/etc/security/cacerts
I/demo   (22262): Root ca path set to /system/etc/security/cacerts
I/demo   (22262): [Platform Helper] getDnsServers() returning 2405:200:800::11, 2405:200:800::11,
I/demo   (22262): [Android Platform Helper] Found DNS server 2405:200:800::11
I/demo   (22262): [Android Platform Helper] Found DNS server 2405:200:800::11
I/demo   (22262): Found Android plugin libmsandroidcamera2.so
I/demo   (22262): Found Android plugin libmswebrtc.so
I/demo   (22262): Found Android plugin libmsaaudio.so
I/demo   (22262): Loading ms plugins from list
I/demo   (22262): Loading plugin libmsandroidcamera2.so...
I/demo   (22262): [Camera2 Capture] libmsandroidcamera2 plugin loaded
I/demo   (22262): Plugin loaded (libmsandroidcamera2.so)
I/demo   (22262): Loading plugin libmswebrtc.so...
I/demo   (22262): libmswebrtc 1.1.1 plugin loaded, iSAC codec version 3.6.0, iLBC codec version 1.1.1
I/demo   (22262): Plugin loaded (libmswebrtc.so)
I/demo   (22262): Loading plugin libmsaaudio.so...
I/demo   (22262): [AAudio] libmsaaudio plugin loaded
I/demo   (22262): [Android Audio Utils] Using 48000 for sample rate value
I/demo   (22262): [AAudio] libaaudio.so successfully loaded
I/AudioEffect(22262): queryNumberEffects(): *numEffects = 17, ret = 0
I/demo   (22262): This device (OnePlus/DN2101/mt6893) declares it has a built-in echo canceller.
I/demo   (22262): No information available for [OnePlus/DN2101/mt6893],
I/demo   (22262): Android >= 8, using sound device descriptor.with DEVICE_HAS_BUILTIN_AEC | DEVICE_HAS_BUILTIN_OPENSLES_AEC flags
I/demo   (22262): Sound device information for [OnePlus/DN2101/mt6893] is: builtin=[yes], delay=[0] ms
I/demo   (22262): [AAudio] Soundcard created
I/demo   (22262): Plugin loaded (libmsaaudio.so)
I/demo   (22262): All plugins in list correctly loaded
W/demo   (22262): Device [Android Sound card] type is unknown
I/demo   (22262): linphone_core_set_playback_gain_db(): no active call.
I/demo   (22262): linphone_core_set_mic_gain_db(): no active call.
I/demo   (22262): Sal nat helper [enabled]
I/demo   (22262): Sal use rports [enabled]
I/demo   (22262): MTU is supposed to be 1300, rtp payload max size will be 1240
I/demo   (22262): Supported codec opus/48000 fmtp=useinbandfec=1 automatically added to codec list.
I/demo   (22262): Could not find encoder for SILK
I/demo   (22262): Supported codec speex/16000 fmtp=vbr=on automatically added to codec list.
I/demo   (22262): Supported codec speex/8000 fmtp=vbr=on automatically added to codec list.
I/demo   (22262): Supported codec PCMU/8000 fmtp= automatically added to codec list.
I/demo   (22262): Supported codec PCMA/8000 fmtp= automatically added to codec list.
I/demo   (22262): Supported codec GSM/8000 fmtp= automatically added to codec list.
I/demo   (22262): Supported codec G722/8000 fmtp= automatically added to codec list.
I/demo   (22262): Supported codec iLBC/8000 fmtp=mode=30 automatically added to codec list.
I/demo   (22262): Could not find encoder for AMR
I/demo   (22262): Could not find encoder for AMR-WB
I/demo   (22262): Could not find encoder for G729
I/demo   (22262): Could not find encoder for mpeg4-generic
I/demo   (22262): Could not find encoder for mpeg4-generic
I/demo   (22262): Could not find encoder for mpeg4-generic
I/demo   (22262): Could not find encoder for mpeg4-generic
I/demo   (22262): Could not find encoder for mpeg4-generic
I/demo   (22262): Supported codec iSAC/16000 fmtp= automatically added to codec list.
I/demo   (22262): Supported codec speex/32000 fmtp=vbr=on automatically added to codec list.
I/demo   (22262): Could not find encoder for SILK
I/demo   (22262): Could not find encoder for SILK
I/demo   (22262): Could not find encoder for SILK
I/demo   (22262): Could not find encoder for G726-16
I/demo   (22262): Could not find encoder for G726-24
I/demo   (22262): Could not find encoder for G726-32
I/demo   (22262): Could not find encoder for G726-40
I/demo   (22262): Could not find encoder for AAL2-G726-16
I/demo   (22262): Could not find encoder for AAL2-G726-24
I/demo   (22262): Could not find encoder for AAL2-G726-32
I/demo   (22262): Could not find encoder for AAL2-G726-40
I/demo   (22262): Could not find encoder for CODEC2
I/demo   (22262): Could not find encoder for BV16
I/demo   (22262): Could not find encoder for 1016
I/demo   (22262): Could not find encoder for G723
I/demo   (22262): Could not find encoder for LPC
I/demo   (22262): Supported codec L16/44100 fmtp= automatically added to codec list.
I/demo   (22262): Supported codec L16/44100 fmtp= automatically added to codec list.
I/demo   (22262): Could not find encoder for CN
I/demo   (22262): Supported codec VP8/90000 fmtp= automatically added to codec list.
I/demo   (22262): Supported codec H264/90000 fmtp=profile-level-id=42801F automatically added to codec list.
I/demo   (22262): Supported codec H265/90000 fmtp= automatically added to codec list.
I/demo   (22262): Could not find encoder for MP4V-ES
I/demo   (22262): Could not find encoder for H263-1998
I/demo   (22262): Could not find encoder for H263
I/demo   (22262): Could not find encoder for H261
I/demo   (22262): Could not find encoder for MPV
I/demo   (22262): Supported codec red/1000 fmtp= automatically added to codec list.
I/demo   (22262): Supported codec t140/1000 fmtp= automatically added to codec list.
I/demo   (22262): Sal use rports [enabled]
I/demo   (22262): belle_sip_stack_set_well_know_port() : set to [5060]
I/demo   (22262): belle_sip_stack_set_well_know_port_tls() : set to [5061]
I/demo   (22262): Root ca path set to /data/user/0/com.example.demok/files/share/linphone/rootca.pem
I/demo   (22262): Invalidating friends maps for list [0xb4000075e0076980]
D/TAG     (22262): outgoingCall: mydomain.com
D/TAG     (22262): outgoingCall: null
I/demo   (22262): New MediaSession [0xb400007588a76ad8] initialized (liblinphone version: 5.2.18)
I/demo   (22262): Media local-ip for streams advertised in SDP: 2409:40c2:c:c6c5:8000::
I/demo   (22262): No account set for this call, using rtp bundle enablement from LinphoneCore.
I/demo   (22262): Rtp bundle is disabled.
I/demo   (22262): Reassigning payload type 101 telephone-event/16000 because already offered
I/demo   (22262): stream#0 [audio] in state [Stopped]: multicast role is [inactive]
I/demo   (22262): RtpSession bound to [::0] ports [48894] [58016]
W/demo   (22262): Fail to increase socket's recv buffer size with SO_RCVBUFFORCE: Operation not permitted.
W/demo   (22262): Fail to increase socket's send buffer size with SO_SNDBUFFORCE: Operation not permitted.
W/demo   (22262): Fail to increase socket's recv buffer size with SO_RCVBUFFORCE: Operation not permitted.
I/demo   (22262): Configured srtp crypto suite: AES_CM_128_HMAC_SHA1_80
I/demo   (22262): Configured srtp crypto suite: AES_CM_128_HMAC_SHA1_32
I/demo   (22262): Configured srtp crypto suite: AES_256_CM_HMAC_SHA1_80
I/demo   (22262): Configured srtp crypto suite: AES_256_CM_HMAC_SHA1_32
I/demo   (22262): Configured srtp crypto suite: AEAD_AES_128_GCM
I/demo   (22262): Configured srtp crypto suite: AEAD_AES_256_GCM
I/demo   (22262): Creating ZRTP engine on rtp session [0xb40000759a634400] ssrc 0x453e0fcf
I/demo   (22262): rtp_session_enable_network_simulation:DISABLING NETWORK SIMULATION
I/demo   (22262): Setting DSCP to 46 for MSAudio stream.
I/demo   (22262): Created stream of type audio at index 0: stream#0 [audio] in state [Stopped]
I/AudioEffect(22262): queryNumberEffects(): *numEffects = 17, ret = 0
I/demo   (22262): This device (OnePlus/DN2101/mt6893) declares it has a built-in echo canceller.
I/demo   (22262): No information available for [OnePlus/DN2101/mt6893],
I/demo   (22262): Android >= 8, using sound device descriptor.with DEVICE_HAS_BUILTIN_AEC | DEVICE_HAS_BUILTIN_OPENSLES_AEC flags
I/demo   (22262): Sound device information for [OnePlus/DN2101/mt6893] is: builtin=[yes], delay=[0] ms
W/demo   (22262): Could not apply gain on sent RTP packets: gain control wasn't activated. Use audio_stream_enable_gain_control() before starting the stream.
I/demo   (22262): [MS2AudioStream] setting type of soundcard 0xb40000758af18640 to voice
I/demo   (22262): [MS2AudioStream] setting type of soundcard 0xb40000758af184b0 to voice
I/demo   (22262): [ToneManager] create ToneManager()
I/demo   (22262): [ToneManager] stopDtmfStream
I/demo   (22262): CallSession [0xb400007588a76ad8] moving from state LinphoneCallIdle to LinphoneCallOutgoingInit
I/demo   (22262): [ToneManager] stopDtmfStream
I/demo   (22262): ICE state is IceStateNotActivated for stream#0 [audio] in state [Stopped]
I/demo   (22262): Contact has not been fixed, stack will do
W/demo   (22262): no host found in this uri
I/demo   (22262): [org.antlr.runtime.MismatchedTokenException]  reason []
E/demo   (22262): fast_header_address parser error for [sip:linphone@:-1]
W/demo   (22262): Cannot create Address, bad uri [sip:linphone@:-1]
W/demo   (22262): Unable to set contact address for session 0xb400007588ad3800 to  as it is not valid
I/demo   (22262): [sip:linphone@[2409:40c2:c:c6c5:8000::]] calling [sip:2002@mydomain.com:5090] on op [0xb4000075dff62480]
I/demo   (22262): get_jni_env(): thread successfuly attached
I/demo   (22262): bellesip_wake_lock_acquire(): Android wake lock [belle-sip transaction(0xb4000075e0076a60)] acquired [ref=0x47e6]
I/demo   (22262): bellesip_wake_lock_acquire(): cast long of wakelock 18406
I/demo   (22262): transaction [0xb4000075e0076a60]: starting transaction background task with id=[47e6].
E/demo   (22262): No listening point matching for [udp://mydomain.com:5090]
E/demo   (22262): belle_sip_client_transaction_send_request(): no channel available
I/demo   (22262): Changing [client] [INVITE] transaction [0xb4000075e0076a60], from state [INIT] to [TERMINATED]
I/demo   (22262): Client INVITE transaction [0xb4000075e0076a60] terminated
I/demo   (22262): [ToneManager] startErrorTone
I/demo   (22262): [ToneManager] playTone
I/demo   (22262): [AAudio Player] Created using device ID: AAudio Earpiece: DN2101 (2)
I/demo   (22262): speex_lib_ctl init with neon ? 1
I/demo   (22262): configuring resampler output to rate=[48000], nchannels=[1]
I/demo   (22262): ms_filter_link: MSFilePlayer:0xb40000758af24500,0-->MSDtmfGen:0xb40000758af24580,0
I/demo   (22262): ms_filter_link: MSDtmfGen:0xb40000758af24580,0-->MSResample:0xb40000758af24680,0
I/demo   (22262): ms_filter_link: MSResample:0xb40000758af24680,0-->MSAAudioPlayer:0xb40000758af24600,0
I/demo   (22262): Initializing speex resampler in mode [voip]
I/demo   (22262): [AAudio Player] Using DTMF mode
I/demo   (22262): [AAudio Player] Using device ID: AAudio Earpiece: DN2101 (2)
I/demo   (22262): [AAudio Player] Player stream configured with samplerate 48000 and 1 channels
I/AAudio  (22262): AAudioStreamBuilder_openStream() called ----------------------------------------
I/demo   (22262): Priority used: 99
I/AudioStreamBuilder(22262): rate   =  48000, channels  = 1, format   = 1, sharing = EX, dir = OUTPUT
I/AudioStreamBuilder(22262): device =      2, sessionId = -1, perfMode = 12, callback: ON with frames = 0, error callback: ON
I/AudioStreamBuilder(22262): usage  =      3, contentType = 4, inputPreset = 0, allowedCapturePolicy = 0
I/AudioStreamBuilder(22262): privacy sensitive = false
I/AudioStreamBuilder(22262): opPackageName = (null)
I/AudioStreamBuilder(22262): attributionTag = (null)
I/demo   (22262): Ring MSTicker priority increased to maximum.
D/AudioStreamBuilder(22262): build() EXCLUSIVE sharing mode not supported. Use SHARED.
D/om.example.demo(22262): PlayerBase::PlayerBase()
D/AudioStreamTrack(22262): open(), request notificationFrames = -8, frameCount = 0
D/AudioTrack(22262): INSP: type=audio,case=1,set=0xb4000075943e8800,stream_type=-1,sample_rate=48000,format=0x1,channel_mask=0x1,frame_count=0,flags=0x104,notification_frames=-8,session_id=0,transfer_type=1,uid=-1,pid=-1
D/AudioTrack(22262): INSP: type=audio,case=1,attributes=1,content_type=4,usage=3,source=0,flags=0x0
D/AudioTrackExtImpl(22262): AudioTrackExtImpl init
D/AudioTrack(22262): set() streamType -1, sampleRate 48000, format 0x1, channelMask 0x1, frameCount 0, flags #104, notificationFrames -8, sessionId 0, transferType 1, uid -1, pid -1 cbf 1
D/AudioTrack(22262): set(): Building AudioTrack with attributes: usage=3 content=4 flags=0x0 tags=[]
D/AudioTrack(22262): set(): 0xb4000075943e8800, Create AudioTrackThread, tid = 28218
I/AudioTrack(22262): createTrack_l(537538608): AUDIO_OUTPUT_FLAG_FAST successful; frameCount 0 -> 2048
D/AudioTrack(22262): createTrack_l(12203): 0xb4000075943e8800, mCblk = 0x770759e000, mLatency = 84, mAfLatency = 42, frameCount = 2048, mSampleRate = 48000, mFlags = 0x4, mReqFrameCount = 2048, mNotificationFramesAct = 256
V/ListServiceUtils(22262): checkInListByUid module:audio-choppy-boost uid:10687 cost 85us return (null)
D/AudioTrack(22262): setVolume(12203): 0xb4000075943e8800, left = 1.000000, right = 1.000000
D/AudioTrack(22262): setVolume left 1.000 right 1.000 , callingPid 22262
D/AAudioStream(22262): setState(s#1) from 0 to 2
D/AudioStreamTrack(22262): open() flags changed from 0x00000104 to 0x00000004
I/AAudio  (22262): AAudioStreamBuilder_openStream() returns 0 = AAUDIO_OK for s#1 ----------------
I/demo   (22262): [AAudio Player] Player stream opened
I/demo   (22262): [AAudio Player] Expected content type 4, got 4
I/demo   (22262): [AAudio Player] Expected usage 3, got 3
D/AAudio  (22262): AAudioStream_requestStart(s#1) called --------------
D/AAudioStream(22262): setState(s#1) from 2 to 3
D/AudioTrack(22262): start(12203): prior state:STATE_STOPPED output 29 stream 8 session 83985
D/AudioTrack(22262): start(12203): 0xb4000075943e8800, prior state:STATE_STOPPED
D/AudioStreamTrack(22262): onAudioDeviceUpdate() audioIo 29 devId 2 => 2
D/AAudio  (22262): AAudioStream_requestStart(s#1) returned 0 ---------
I/demo   (22262): [AAudio Player] Player stream started
I/demo   (22262): Playing tones of frequencies 620,0 Hz, duration=250, amplitude=1.000000 interval=250, repeat_count=3
I/demo   (22262): CallSession [0xb400007588a76ad8] moving from state LinphoneCallOutgoingInit to LinphoneCallError
I/demo   (22262): StreamsGroup::finish() called.
I/demo   (22262): =================================================================================
I/demo   (22262):                          FILTER USAGE STATISTICS
I/demo   (22262): Name                          Count           Time/tick (ms)            CPU Usage
I/demo   (22262):                                         min     mean    max     sd
I/demo   (22262): ---------------------------------------------------------------------------------
I/demo   (22262): MSResample                    0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): MSAAudioPlayer                0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): MSDtmfGen                     0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): MSFilePlayer                  0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): MSWebRTCAECM                  0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): MSRtpSend                     0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): =================================================================================
I/demo   (22262): =================================================================================
I/demo   (22262):                          FILTER USAGE STATISTICS
I/demo   (22262): Name                          Count           Time/tick (ms)            CPU Usage
I/demo   (22262):                                         min     mean    max     sd
I/demo   (22262): ---------------------------------------------------------------------------------
I/demo   (22262): MSResample                    0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): MSAAudioPlayer                0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): MSDtmfGen                     0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): MSFilePlayer                  0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): MSWebRTCAECM                  0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): MSRtpSend                     0         0.00    0.00    0.00    0.00          0.0
I/demo   (22262): =================================================================================
I/demo   (22262): Stopping ZRTP context on session [0x0]
I/demo   (22262): ZRTP context destroyed
I/demo   (22262): Resetting the current call
I/demo   (22262): Removing the call (local address sip:linphone@[2409:40c2:c:c6c5:8000::] remote address sip:2002@mydomain.com:5090) from the list attached to the core
I/demo   (22262): CallSession [0xb400007588a76ad8] moving from state LinphoneCallError to LinphoneCallReleased
D/AudioStreamTrack(22262): onAudioDeviceUpdate() audioIo 29 devId 2 => 2
I/demo   (22262): op [0xb4000075dff62480] : setOrUpdateDialog() current=[0x0] new=[0x0]
I/demo   (22262): Destroying op [0xb4000075dff62480] of type [SalOpCall]
I/demo   (22262): transaction [0xb4000075e0076a60]: ending transaction background task with id=[47e6].
I/demo   (22262): bellesip_wake_lock_release(): Android wake lock released [ref=0x47e6]
I/demo   (22262): Transaction [0xb4000075e0076a60] deleted
I/Quality (22262): Skipped: false 21 cost 352.66943 refreshRate 0 processName com.example.demok
I/flutter (22262): Call success : Flutter
I/demo   (22262): [Push Notification] Token fetched from Firebase: e2eGrkm_SKKG1RKRNqe9kv:APA91bGC7RCNGStkhwdzl3Gi01lCESDcy5rksLn96u6Ah7Cp_8AvnJmJnzuuRuMra4Yv_fDdxTyhL1Is93Yyf6dn3Hf63z7k7tEA6jWTj-g-1xgO9BG1fv51d9WW4aRH7r8qLm8RKBcL
I/demo   (22262): [Core Manager] Push notification app id is [94851214726] and token is [e2eGrkm_SKKG1RKRNqe9kv:APA91bGC7RCNGStkhwdzl3Gi01lCESDcy5rksLn96u6Ah7Cp_8AvnJmJnzuuRuMra4Yv_fDdxTyhL1Is93Yyf6dn3Hf63z7k7tEA6jWTj-g-1xgO9BG1fv51d9WW4aRH7r8qLm8RKBcL]
D/AAudioStream(22262): setState(s#1) from 3 to 4

**Comment by PrarthanaBhatt** [comment]
Hi,
I tried to add core.interpretUrl(remoteSipUri) 
but still getting 

E/demo: fast_header_address parser error for [sip:linphone@:-1]
W/demo: Cannot create Address, bad uri [sip:linphone@:-1]

E/demo: No listening point matching for [udp://mydomain.com:5090]
E/demo: belle_sip_client_transaction_send_request(): no channel available

**Comment by Viish** [comment]
@PrarthanaBhatt that means you don't have a default account. You can always use the full URI like `Factory.instance().createAddress("sip:" + username + "@" + domain)`

**Comment by PrarthanaBhatt** [comment]
Hi @Viish,
I have tried with the same format but still not getting.

Factory.instance().createAddress("sip:" + username + "@" + domain)

E/demo: fast_header_address parser error for [sip:linphone@:-1]
W/demo: Cannot create Address, bad uri [sip:linphone@:-1]

E/demo: No listening point matching for [udp://mydomain.com:5090]
E/demo: belle_sip_client_transaction_send_request(): no channel available

we are tried using this two versions of sdk,
   linphone-sdk-android-5.1.38.aar,
   linphone-sdk-android-5.2.18.aar

**Comment by kukadiyaAni** [comment]
@PrarthanaBhatt I got same error did you find any solution for that
